File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.*;

class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] n = new int[3];
        int sum = 0;
        for(int i=0; i<3; i++) {
            n[i] = sc.nextInt();
            sum += n[i];
        }

        if(n[0]==60 && n[1]==60 && n[2]==60)
            System.out.println("Equilateral");
        else if(sum == 180 && (n[0]==n[1] || n[0]==n[2] || n[1]==n[2]))
            System.out.println("Isosceles");
        else if(sum == 180)
            System.out.println("Scalene");
        else
            System.out.println("Error");
        sc.close();
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.kt
fun main() {
    val angle = List(3) { readln() }.map { it.toInt() }
    if (angle.all { it == 60 }) {
        println("Equilateral")
        return
    }
    if (angle.sum() != 180) {
        println("Error")
        return
    }
    if (angle.sum() == 180 && angle.size == angle.distinct().size) {
        println("Scalene")
        return
    }
    println("Isosceles")
}
